src/sklearn_evaluation/plot/regression.py

```diff
@@ -103,24 +101,35 @@ def prediction_error(y_true, y_pred, model=None, ax=None):
     ax: matplotlib Axes
         Axes containing the plot
 
+    Notes
+    -----
+    .. deprecated:: 0.8.6
+        'model' argument is deprecated, will be removed in version 0.8.8
+
     Examples
     --------
     .. plot:: ../examples/prediction_error.py
 
     """
 
+    deprecated.parameter_deprecated(
```

Comment:
Please add a test for this (https://docs.pytest.org/en/7.1.x/how-to/capture-warnings.html#warns)

also, the deprecation will happen in two major versions to that'd be `0.10`, I think since we recently release this, let's remove it in 0.9

Reply:
+ We've added links to the `intro.md` page, include this guide as well in it.

Reply:
deprecation is fine 0.8.6, removal in 0.9

Reply:
Added in intro.md @idomic